ui/app/models/config-ui/message.js

```diff
@@ -11,10 +11,21 @@ import { withFormFields } from 'vault/decorators/model-form-fields';
 const validations = {
   title: [{ type: 'presence', message: 'Title is required.' }],
   message: [{ type: 'presence', message: 'Message is required.' }],
+  link: [
+    {
+      validator(model) {
+        if (!model?.link) return true;
+        const linkTitle = Object.keys(model.link).length === 1;
+        const linkHref = Object.values(model.link);
+        return linkTitle && !!linkHref[0];
```

Comment:
You could use descructuring here
```suggestion
        const [linkHref] = Object.values(model.link);
        return linkTitle && !!linkHref;